src/com/pentalog/pentastagiu/homework2/UserApp/Board.java

```diff
@@ -0,0 +1,30 @@
+package com.pentalog.pentastagiu.homework2.UserApp;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Board {
+    private static int NumberOfMessages;
+    private static  List<String> AllMesages = new ArrayList<>();
+    Messages messages;
+
+    public void AddMessage(Messages messages){
+        AllMesages.add(messages.getDescription());
+        AllMesages.add(messages.getAuthor());
+        NumberOfMessages++;
+    }
+    public void DispalyMessages()
+    {
+        String arr[]= new String[AllMesages.size()];
```

Comment:
sa incerci sa inlocuieste logica folosind direct lista de mesaje pe care ai creat-o, nu e nevoie sa mai creezi un array; poti folosi tot un for in care i-ul creste din 2 in 2 si sa concatenezi allMessages.get(i) + " wrote by " + allMessages.get(i+1) sau daca in Message, in loc de field-ul author care e acum definit ca String, daca ai avea un User, atunci nici nu ar mai fi nevoie sa adaugi in AllMesages si descrierea si autorul: ai avea o lista de <Message> si cand e nevoie sa afisezi mesajele vei itera pe mesajele din lista (for (Message message: messages) si ai formula direct mesajul pe care vrei sa il afisezi: message.getDescription() + " wrote by " + message.getAuthor().getEmail()